internal/store/pod.go

```diff
@@ -986,6 +986,42 @@ var (
 					}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_pod_overhead",
+			Type: metric.Gauge,
+			Help: "The pod overhead associated with running a pod.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if p.Spec.Overhead != nil {
+					for resourceName, val := range p.Spec.Overhead {
+
+						switch resourceName {
+						case v1.ResourceCPU:
+							ms = append(ms, &metric.Metric{
+								LabelValues: []string{p.Name, p.Spec.NodeName, sanitizeLabelName(string(resourceName)), string(constant.UnitCore)},
+								Value:       float64(val.MilliValue()) / 1000,
+							})
+
+						case v1.ResourceMemory:
+							ms = append(ms, &metric.Metric{
+								LabelValues: []string{p.Name, p.Spec.NodeName, sanitizeLabelName(string(resourceName)), string(constant.UnitByte)},
+								Value:       float64(val.Value()),
+							})
+						}
+					}
+
+				}
+
+				for _, metric := range ms {
+					metric.LabelKeys = []string{"pod", "node", "resource", "unit"}
```

Comment:
`pod` is a default label that will be added to all pod metrics.
```suggestion
					metric.LabelKeys = []string{"node", "resource", "unit"}
```

Reply:
Please remove the instances of pod label values in your new metric as well.

Reply:
ACK - sounds good.